engine-vuex/src/store.ts

```diff
@@ -117,6 +117,35 @@ export class ImagesetInfo {
   }
 }
 
+export function isImageSetInfo(o: any): o is ImagesetInfo { // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I think that we don't need this function? I'm pretty sure that the relevant tests can be done with the TypeScript `instanceof` operator. The key difference versus some of our other code is that `ImagesetInfo` is a class, while the various `is*` functions are for interfaces, or classes defined in the ScriptSharp engine code where it's not clear that the `instanceof` operator will do the right thing. But here, I'm pretty sure that `instanceof` will meet our needs.